common/net/http/headers.go

```diff
@@ -1,6 +1,7 @@
 package http
 
 const (
+	headerMidazID       = "Midaz-ID"
```

Comment:
what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.